fe/src/project/Seller.Portal/areas/Clients/pages/GroupPage/GroupPage.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { toast } from "react-toastify";
+import { ThemeProvider } from "@material-ui/core/styles";
+import GlobalHelper from "../../../../../../shared/helpers/globals/GlobalHelper";
+import Header from "../../../../../../shared/components/Header/Header";
+import Store from "../../../../../../shared/stores/Store";
+import Footer from "../../../../../../shared/components/Footer/Footer";
+import MenuTiles from "../../../../../../shared/components/MenuTiles/MenuTiles";
+import GroupForm from "../../components/GroupForm/GroupForm";
+
+const GroupPage = (props) => {
+
+  toast.configure();
+
+  return (
+    <ThemeProvider theme={GlobalHelper.initMuiTheme()}>
+      <Store>
+        <div>
+          <Header {...props.header}></Header>
+          <MenuTiles {...props.menuTiles} />
+          <GroupForm {...props.groupForm} />
```

Comment:
suggestion: Rename to <ClientGroupForm {...props.clientGroupForm} /> like everywhere else